### ISSUE TITLE
bug: shift select is off by 1

### ISSUE THREAD

**Issue by davidkircos** [quadratichq/quadratic#1375]
![CleanShot 2024-05-21 at 20 40 35](https://github.com/quadratichq/quadratic/assets/3479421/eb77f866-8242-4419-ad6c-e7d7c9353f2a)

**Comment by davidfig** [comment]
This is correct as it only checks the cursor cell (the same as excel). There are some bugs with shift+cmd+arrow around code cells, though.

**Comment by davidkircos** [comment]
In the GIF above, I press command + shift + left arrow + up arrow. It seems like it should just select the other content, not the blank space.
